javascript/javascript1/week2/week2-guzide/student-manager/StudentManager.js

```diff
@@ -0,0 +1,37 @@
+const class07Students = [];
+
+function getNumberOfStudents() {
+  return class07Students.length;
+}
+
+function addStudentToClass(studentName) {
+  if (studentName == "") {
+    console.log("You can't leave this area empty");
+    return;
+  }
+
+  if (studentName === "Queen Mary") {
+    if (!class07Students.includes(studentName)) {
+      class07Students.push(studentName);
+      console.log(class07Students);
+    }
+    return;
+  }
+
+  if (getNumberOfStudents() >= 6) {
+    console.log("Cannot add more students to class 07");
+    return;
+  }
+  class07Students.push(studentName);
+  console.log(class07Students);
+}
+
+addStudentToClass("");
+addStudentToClass("Güzide");
```

Comment:
You have a bug in your code 🐛 
You need to move one if statement to make it work properly. Try adding the same name twice to the students array.

Reply:
Issue is resolved. Thank you.